user/articles/3-layout-and-design/12-widget-reference.markdown

```diff
@@ -1,41 +1,30 @@
 # Commerce Widgets
```

Comment:
I expect the reader to use this page as a reference, meaning it needs to be formatted like a reference doc. That's the reason for removal of the images. I'd also like to see it formatted as either a single large table or a few tables with a categorical grouping of widgets. See the portlet list doc tables as a sort of guide: https://sync.liferay.com/web/marketing/home?p_p_id=com_liferay_document_library_web_portlet_DLPortlet&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_document_library_web_portlet_DLPortlet_mvcRenderCommandName=%2Fdocument_library%2Fview_file_entry&_com_liferay_document_library_web_portlet_DLPortlet_fileEntryId=2853166